core/src/com/unciv/ui/worldscreen/WorldScreen.kt

```diff
@@ -502,5 +504,13 @@ class WorldScreen(val viewingCiv:CivilizationInfo) : CameraStageBaseScreen() {
             displayTutorial(Tutorial.Embarking)
     }
 
+    private fun exitGamePrompt() {
+        var promptWindow = PopupTable(this)
+        promptWindow.add("Do you want to exit the game?".tr())
+        promptWindow.row()
+        promptWindow.addButton("Yes"){game.exitEvent?.invoke()}
+        promptWindow.addButton("No") {promptWindow.close()}
```

Comment:
Missing localization: `"Yes"` to `"Yes".tr()` (same for No)

Reply:
Good comment, however, it is not needed there to avoid calling it twice.
It is already called within a `addButton `as `TextButton(text.tr(), skin).apply`.